scripts/bootstrap.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# For local development we need this script tp create collections in a local jamdb.
+# For deployment this will need to be run manaually.
+
+source ~/.bashrc
+
+workon `lsvirtualenv -b |grep jam`
```

Comment:
Acknowledging that yes, this script does have shortcomings:

1. Use virtualenvwrapper. It's better. But please feel free to add whatever commands you need to activate your virtualenv, even if these fail for other developers it shouldn't break the script execution. 
2. Not a regression on the old script, but I can update this to first drop the namespace
3. Really? My local tests suggested that the exit code of `source` didn't effect the script execution.
4. Yes, we might need to add this,.